.github/workflows/pr-benchmark-generate.yml

```diff
@@ -83,20 +84,23 @@ jobs:
         #       TO the start of the branch history.
         #       With it, we run benchmarks ONLY FOR the latest commit.
         echo "Running benchmarks for upstream/master..."
-        DJC_BENCHMARK_QUICK=1 asv run upstream/master^! -v
+        DJC_BENCHMARK_QUICK=1 asv run upstream/master^! -v --machine ${MACHINE}
         echo "Benchmarks for upstream/master DONE."
         echo "Running benchmarks for HEAD..."
-        DJC_BENCHMARK_QUICK=1 asv run HEAD^! -v
+        DJC_BENCHMARK_QUICK=1 asv run HEAD^! -v --machine ${MACHINE}
         echo "Benchmarks for HEAD DONE."
 
-        # Compare against master
-        echo "Comparing benchmarks..."
+        echo "Creating pr directory..."
         mkdir -p pr
-        asv compare upstream/master HEAD --factor 1.1 --split > ./pr/benchmark_results.md
-
         # Save the PR number to a file, so that it can be used by the next step.
         echo "${{ github.event.pull_request.number }}" > ./pr/pr_number.txt
 
+        # Compare against master
+        # NOTE: The command is run twice, once so we can see the debug output, and once to save the results.
+        echo "Comparing benchmarks... (debug)"
+        asv compare upstream/master HEAD --factor 1.1 --split --machine ${MACHINE} --verbose
```

Comment:
The bug was that the `asv compare` command threw error as it needed to be specified which data set to use. I fixed this by specifying the `--machine`.

This problem popped up after the release of v0.131, which is the first release that is actually benchmarked. So after v0.131 was released, we saved the benchmark data for that release, and it means that, when afterwards the PR benchmarking workflow ran, it all of a sudden found new benchmarking datasets in the `.asv` directory. And so the `asv compare` command didn't know which one to use, and needed to be explicitly said.